### ISSUE TITLE
Sortier fails if typescript isn't installed

### ISSUE THREAD

**Issue by k2snowman69** [snowcoders/sortier#848]
Sortier seem fails to run because typescript is missing
```
D:\Projects\sortier\issue811>yarn run sortier
yarn run v1.22.4
$ D:\Projects\sortier\issue811\node_modules\.bin\sortier
internal/modules/cjs/loader.js:983
  throw err;
  ^

Error: Cannot find module 'typescript'
Require stack:
- D:\Projects\sortier\issue811\node_modules\@typescript-eslint\typescript-estree\dist\parser.js
```

**Comment by k2snowman69** [comment]
Fixed in #811 and will be released in the next patch or higher

**Comment by k2snowman69** [comment]
Verified fixed
